### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
Made the following changes:
- Delete App.css
- Delete all the text, images, and styles form CRA
- Create components directory
- Create a new Calculator.js file in components directory
- Develop calculator UI

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                
+                <section id="buttons">
+                    <div id="screen">0</div>
+                    <button>AC</button>
+                    <button>+/-</button>
+                    <button>%</button>
+                    <button className="last-row">/</button>
+                    
+                    
+                    <button>7</button>
```
Comment (rloterh) [inline src/components/Calculator.js]: - [ ] OPTIONAL
Kindly ensure not to leave empty lines in your code which are not serving any purpose, so as to keep your code well organized.

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import calculator from './components/Calculator';
 
 function App() {
```
Comment (rloterh) [inline src/App.js]: - [x] The main component `App.js` is stateless. Kindly change the component to a **class/ stateful** component

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import calculator from './components/Calculator';
```
Comment (rloterh) [inline src/App.js:8]: - Kindly delete `logo.svg` and other unused files so as to keep your project structure clean.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Hamid, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Stateful component successfully created
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @hamid-murambiwa 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**